buildroot/share/PlatformIO/scripts/SAMD21_minitronics20.py

```diff
@@ -6,7 +6,8 @@
 if pioutil.is_pio_build():
     from os.path import join, isfile
     import shutil
-
+    from SCons.Script import DefaultEnvironment
+    env = DefaultEnvironment()
     Import("env")
```

Comment:
I believe this line becomes redundant when we go around PlatformIO to get `env` directly from SCons. Another way to do this is `import marlin` which automatically defines `env` as a global. But if the only aim here is to eliminate a complaint from IntelliSense then it may be enough to just put `env = None` in front of `Import("env")`.